src/sagas/helpers.js

```diff
@@ -1,5 +1,31 @@
 import { get } from 'lodash';
-import { put, call, race, take } from 'redux-saga/effects';
+import {
+  put,
+  call,
+  race,
+  take,
+  select,
+} from 'redux-saga/effects';
+import { types } from '../actions';
+import { FEATURE_TOGGLE_DEFAULTS } from '../constants';
+
+export function* waitForFeatureToggleInitialization() {
+  const featureTogglesInitialized = yield select((state) => state.featureTogglesInitialized);
+
+  if (!featureTogglesInitialized) {
+    // Wait until featureToggle saga completed initialization, which includes
+    // downloading the current toggle status for this client.
+    yield take(types.FEATURE_TOGGLE_INITIALIZED);
+  }
+}
+
+export function* checkForFeatureFlag(flag) {
```

Comment:
Missing docstring

Reply:
Added in 391762d466c93306eb5a706ac7ebf923994bc80e

Reply:
Missing the return type